src/test/resources/application-test.properties

```diff
@@ -3,3 +3,4 @@ spring.datasource.platform=h2
 spring.datasource.driver-class-name=org.h2.Driver
 spring.datasource.url=jdbc:h2:mem:db;DB_CLOSE_DELAY=-1
 spring.jpa.hibernate.ddl-auto=create
+spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
```

Comment:
Why do we need this?

Reply:
Otherwise I get the following Exception in the test `<org.hibernate.LazyInitializationException: could not initialize proxy [science.icebreaker.account.Account#3] - no Session>`